src/demo/pages/DemoContainer.jsx

```diff
@@ -2182,6 +2211,91 @@ class DemoContainer extends React.Component {
                 </div>
               )}
             />
+            <Documentation
+              name="Verical Scroll View with auto scroll"
+              component={(
+                <>
+                  <div className="mb-5">
+                    <Button
+                      label="Add text into scroll view"
+                      clickHandler={
+                        () => {
+                          this.setState((prevState) => ({
+                            scrollView1: `${prevState.scrollView1}  ${this.constructor.generateRandomString()}`,
+                          }));
+                        }
+                      }
+                    />
+                    <CheckboxField
```

Comment:
`Radio` would be better here — now I'm really unsure what effect the checkbox turns on 🙂.

![image](https://user-images.githubusercontent.com/5614085/92750087-064c0000-f387-11ea-840a-96b8bde687cb.png)